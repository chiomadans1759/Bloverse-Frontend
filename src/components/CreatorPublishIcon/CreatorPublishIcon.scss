@import 'sass-rem';

#bl-creator-add {
    position: fixed;
    right: 2.5rem;
    bottom: 4.5rem;
  .button-container {
    background-color: #2F80ED;
    height: rem(38px);
    width: rem(38px);
    border-radius: 50%;
    .icon {
      position: absolute;
      left: 50%;
      top: 50%;
      transform: translate(-50%, -50%);
      color: #ffffff;
    }
  }
}

#bl-creator-add:hover {
  cursor: pointer;
}
