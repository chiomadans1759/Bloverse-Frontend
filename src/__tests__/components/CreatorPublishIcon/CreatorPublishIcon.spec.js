import { mount } from '@vue/test-utils'
import CreatorPublishIcon from '@/components/CreatorPublishIcon/CreatorPublishIcon.vue';

describe('CreatorPublishIcon Component', () => {
  it('should render correctly', () => {

    const wrapper = mount(CreatorPublishIcon , {});
    expect(wrapper.element).toMatchSnapshot();
  });
});
