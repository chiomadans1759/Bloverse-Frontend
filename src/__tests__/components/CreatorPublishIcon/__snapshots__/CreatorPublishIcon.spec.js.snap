// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreatorPublishIcon Component should render correctly 1`] = `
<main
  id="bl-creator-add"
>
  <div
    class="button-container text-center"
  >
    <i
      class="fas fa-plus icon"
    />
  </div>
</main>
`;
